nefertari/authentication/models.py

```diff
@@ -159,6 +159,9 @@ def random_uuid(**kwargs):
 def encrypt_password(**kwargs):
     """ Crypt :new_value: if it's not crypted yet. """
     new_value = kwargs['new_value']
+    if kwargs['event'].is_auth_view:
+        return new_value
+
```

Comment:
not sure to understand what's going on here. why would we return new_value here if this is_auth_view is True? shouldn't it be encrypted first?